packages/core/src/config/crf.ts

```diff
@@ -60,5 +60,11 @@ export const validateSelectedCrfAndCodecCombination = (
 };
 
 export const getActualCrf = (codec: Codec) => {
-	return getCrfOrUndefined() ?? getDefaultCrfForCodec(codec);
+	try {
+		const crf = getCrfOrUndefined() ?? getDefaultCrfForCodec(codec);
+		validateSelectedCrfAndCodecCombination(crf, codec);
+		return crf;
+	} catch (e) {
+		return getDefaultCrfForCodec(codec);
```

Comment:
If an invalid crf for the codec is stored just return the default crf for the codec.